workers/unity/spatialos.AndroidClient.worker.json

```diff
@@ -0,0 +1,69 @@
+{
+  "build": {
+    "tasks": [
+      {
+        "name": "Codegen",
+        "steps": [{"name": "** Load Unity to generate code **", "command": "echo", "arguments": ["No-op."]}]
+      },
+      {
+        "name": "build",
+        "steps": [{"name": "** Run ci/build-client.sh instead **", "command": "echo", "arguments": ["No-op."]}]
```

Comment:
This is inaccurate, `ci/build-client.sh` only builds the `UnityClient` worker. 

Same in the `IosClient` one

Reply:
(y)